dataland-frontend/tests/e2e/specs/dev/UploadDataTest.ts

```diff
@@ -26,9 +34,16 @@ describe('User interactive tests for Data Upload', () => {
         })
     })
 
-    it('Create EU Taxonomy Dataset with Reporting Obligation and Check the Link', () => {
+    it('Create a Company when the login is wrong', () => {
+        cy.restoreLoginSession()
```

Comment:
Entspricht dann das leer lassen der Argumente von `restoreLoginSession()` dem Nicht-Einloggen?

Reply:
Wenn du in die Funktion schaust dann wird als default der data_reader gesetzt